gadget/update.go

```diff
@@ -433,6 +489,13 @@ type DiskVolumeValidationOptions struct {
 	// AllowImplicitSystemData has the same meaning as the eponymously named
 	// filed in EnsureLayoutCompatibilityOptions.
 	AllowImplicitSystemData bool
+	// ExpectedEncryptedPartitions is the a map of the names (gadget structure
```

Comment:
nitpick: sorry for not noticing this before, but there's a "the a" in there

Reply:
fixed thanks for catching this